src/app/src/components/NavBar.js

```diff
@@ -73,10 +74,25 @@ function SettingsButton({ variant }) {
 function ExitButton({ variant }) {
     const navigate = useNavigate();
     const dispatch = useDispatch();
+    const user = useSelector(state => state.auth.user);
     const { '*': params } = useParams();
 
     const boundaryId = params.startsWith('draw/') ? params.substring(5) : '';
 
+    const queries = useSelector(state => state.api.queries);
+    let boundary = null;
+    if (`getBoundaryDetails("${boundaryId}")` in queries) {
+        boundary = queries[`getBoundaryDetails("${boundaryId}")`].data;
+    }
```

Comment:
Cool! I did not know you could do this.

Reply:
Seems dangerous, going with a simpler approach in [d654933](https://github.com/azavea/iow-boundary-tool/pull/156/commits/d654933b711848e16498b04a993bad8ddeef127f)